tests/karate/src/test/java/be/constants.feature

```diff
@@ -205,6 +205,17 @@ Feature: Constants
         }
       """
 
+    # In the specification there is a Pubkey that will never be used by am attendee so we use
+    # it as a supreme source of digital coin distribution (will maybe change later)
+    * def createIssuancePubKey = function(){ return "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=" }
+    # The signature is constant since the backend will not check it either way, it is the front end's
+    # job to perform is check so we can leave it as a constant value
```

Comment:
The backend **currently** don't check it. This is not a specification, and it might change in the future.

I don't expect you to apply this in the PR, but I would prefer if the comment explained that.

Reply:
Agreed